.github/workflows/testAndPublish.yml

```diff
@@ -159,9 +159,19 @@ jobs:
       run: ci/scripts/setBuildVersionVars.ps1
     - name: Set scons args
       run: ci/scripts/setSconsArgs.ps1
-    - name: Prepare source code for launcher
+    - name: Install SignPath
+      if: ${{ env.feature_signing }}
+      run: Install-Module -Name SignPath -Force
+    - name: Create unsigned launcher
+      if: ${{ !(github.event_name == 'push' && env.feature_signing) }}
       shell: cmd
       run: scons %sconsOutTargets% %sconsArgs% ${{ !runner.debug && '--all-cores' || '-j1'}}
+    - name: Create signed launcher
+      if: ${{ github.event_name == 'push' && env.feature_signing }}
+      shell: cmd
+      env:
```

Comment:
Can't we set this conditionally so that we don't need two job steps that run exactly the same command? What happens if we set an environment variable to an empty or nonexistent secret?

Reply:
I think this is much more legible than the alternative. What is your proposed alternative code?

Reply:
GPt suggests:
```suggestion
      env:
        apiSigningToken: ${{ (github.event_name == 'push' && env.feature_signing) && secrets.API_SIGNING_TOKEN || '' }}
```